src/SFA.DAS.AdminService.Web/Controllers/AccountController.cs

```diff
@@ -40,9 +40,17 @@ public IActionResult PostSignIn()
                 return RedirectToAction("InvalidRole", "Home");
             }
 
-            if (HttpContext.User.HasRoatpRoleOnly())
+            if (HttpContext.User.HasRoatpRolesOnly())
```

Comment:
Please chop the `HttpContext` bit so it's consistent with the `User` calls below

Reply:
👍